assignment.md

```diff
@@ -11,63 +11,76 @@ If the version control concepts or `git` commands are still confusing to you, co
         ```
 
     - b. Who was the last person to modify `README.md`? (Hint: use `git log` with an argument)
         ```bash
-        # git command and output here
+        # git log -1 README.md
+      output: commit 7897d58f9a9334bcc55a373734c6a0c5a765d95a Author: Xihe Xie <axiezai@gmail.com>Date:   Sat Feb 6 11:21:02 2021 -0500
         ```
     
     - c. What was the commit message associated with the last modification to the `-Week 2` line of `index.md` file? (Hint: use `git blame` and `git show`, check out their help pages if you are not sure what to do)
+      >"added ch2 assignment" 
 
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > Git stash saves changes I made locally elsewhere that's not on my repo, and it restores my repository to the last commit, any uncommitted changes I had were reversed when the file was opened again
      - b. What do you see when you run `git log --all --oneline` after stashing your modification?
-       > Answer here
+       > 3a13bb9 (refs/stash) WIP on master: 540a90d added week5 
      - c. Look up what `git stash drop` and `git stash pop` does, either Google or use `--help`. Run `git stash drop` to undo what you did with `stash`, in what scenario might `stash`-ing, `pop`-ing, and `drop`-ing be useful?
-       > Answer here
+       > Pop less you make the stashed changes to multiple brances and drop lets you delete any of the stashes you no longer need. Stash is helpful when you're working on multiple files and branches and need to switch content but don't neccesarily want to commit the changes. you can shelf them and apply them later.
```

Comment:
`pop` applies your stashed changes AND throws it away from stash. while drop literally drops your stashed changes so they no longer exist in your git tracker. But good job.